src/modules/cssTweaks.js

```diff
@@ -25,6 +28,24 @@ const DEFAULT_SETTINGS = {
 			type: SETTING_TYPES.BOOLEAN,
 			value: false
 		},
+		[SETTING_IDS.HIDE_LEFT_SIDEBAR_BUTTON]: {
+			name: 'Hide the "< Show" button',
+			description: 'Make the little floating thing disappear',
+			type: SETTING_TYPES.BOOLEAN,
+			value: true
```

Comment:
I'd suggest defaulting this to false. I understand that the sort of person most likely to use AkunX would probably prefer not to see it, but if someone wasn't aware of what it is/does but did want to use the sidebar they'd have to struggle to find the option to show it in the story read menu thing instead.